src/mixer.erl

```diff
@@ -137,7 +137,7 @@ expand_mixin(Line, Name) when is_atom(Name) ->
             io:format(
                 "~s:~p Unable to resolve imported module ~p~n",
                 [get_file_name(), Line, Name]),
-            exit({error, undef_mixin_module});
+            error({error, undef_mixin_module});
```

Comment:
@elbrujohalcon, let me know if this is acceptable. I left a few notes on the Conversation on why I think this is a proper fix.

Reply:
Yeah! It's cool.